services/storage/cfg/config.yml

```diff
@@ -0,0 +1,81 @@
+# Logger section
+logger:
+  level: debug # Minimum enabled logging level
+
+# Profiler section
+pprof:
+  enabled: true
+  address: :6060  # Server address
+  shutdown_timeout: 15s  # Timeout to shut down the HTTP server
```

Comment:
`Timeout to shut down the HTTP server` sounds confusing a bit, maybe replace it with `# timeout for profiling HTTP server graceful shutdown`?

Reply:
almost every comment was taken from the service's config examples OR from the previous `.env` config file

synced SN and IR with your suggestion